examples/eris/src/commands/ping.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: "ping",
+  description: "Pings the bot",
+  async execute(message, args) {
+    const msg = await message.channel.createMessage("Pinging...");
+    const ping = msg.timestamp - message.timestamp;
+    msg.edit("🏓 Pong! `" + ping + "ms`");
```

Comment:
```suggestion
    msg.edit(`🏓 Pong! \`${ping} ms\``);
```

Use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).